mve_intrinsics/mve.template.md

```diff
@@ -11,7 +12,6 @@ toc: true
 This token is necessary to generate the Table of Contents via Jekyll. --->
 * TOC
 {{:toc}}
```

Comment:
Why these do not show up in the PDF?

Reply:
I used sed in the generate-pdf.sh script.